test/with_dummyserver/test_socketlevel.py

```diff
@@ -137,6 +143,38 @@ def socket_handler(listener: socket.socket) -> None:
                 ), "missing ALPN protocol in SSL handshake"
 
 
+def original_ssl_wrap_socket(
```

Comment:
Why call it original? Is the idea that we continue using the original interface of `ssl_wrap_socket`?

Reply:
You already have a function called ssl_wrap_socket but it's part of the system under test. Happy to take suggestions on the name